packages/ui/src/bounty/components/TileSection.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { FundedRange } from '@/bounty/components/FundedRange'
+import { BlockDurationStatistics, StatisticItem, Statistics } from '@/common/components/statistics'
+import { TextHuge, TokenValue } from '@/common/components/typography'
+import { MemberInfo } from '@/memberships/components'
+import { Member } from '@/memberships/types'
+
+export interface TileSectionProps {
+  stage?: string
+  labelTitle?: string
+  tooltipText?: string
+  durationTitle?: string
+  value?: number
+  bountyCreator?: string
+  oracle?: string
+  member: Member
+  cherryLabel?: string
+  cherryValue?: number
+  cherryTooltipText?: string
+  entrantLabel?: string
+  entrantTooltipText?: string
+  entrantValue?: number
+}
+
+export const TileSection = React.memo(
+  ({
+    stage,
+    labelTitle,
+    tooltipText,
+    durationTitle,
+    value,
+    bountyCreator,
+    oracle,
+    member,
+    cherryLabel,
+    cherryTooltipText,
+    cherryValue,
+    entrantLabel,
+    entrantValue,
+    entrantTooltipText,
+  }: TileSectionProps) => {
+    return (
+      <>
+        <Statistics>
+          <StatisticItem title={labelTitle} tooltipText={tooltipText}>
+            <TextHuge>{stage}</TextHuge>
```

Comment:
`TextHuge` with `{stage}` should have `font-family: ${Fonts.Grotesk};`
![image](https://user-images.githubusercontent.com/47782567/146224896-4aa8e613-9da9-4bdb-8775-9cbdcb27c496.png)